Specify account suspension (#2014)
matrix-org/matrix-spec: content/client-server-api/_index.md

To enable users to appeal to a lock clients MAY use
[server contact discovery](#getwell-knownmatrixsupport).

#### Account suspension

{{% added-in v="1.13" %}}

> Similar to [locking](#account-locking), a user's account MAY be suspended to
> prevent further activity from that account. A similar effect is achieved, though

Reviewer: I found the "A similar effect is achieved" part hard to grok. Also, avoid passive voice.

```suggestion
Server administrators MAY suspend a user's account to prevent further activity
from that account. The effect is similar to [locking](#account-locking), though
```

without risk of the client losing state from a logout. Suspensions are reversible,
like locks and unlike deactivations.

The actions a user can perform while suspended is deliberately left as an
implementation detail. Servers SHOULD permit the user to perform at least the
following, however:

* Log in and create additional sessions (which are also suspended).
* See and receive messages, particularly through [`/sync`](#get_matrixclientv3sync)
  and [`/messages`](#get_matrixclientv3roomsroomidmessages).
* [Verify other devices](#device-verification) and write associated
  [cross-signing data](#cross-signing).
* [Populate their key backup](#server-side-key-backups).
* Leave rooms and reject invites.

> * Redacting their own events.

Reviewer: ```suggestion
* Redact their own events.
```

* Log out or delete any device of theirs, including the current session.
* Deactivate their account, potentially with a time delay to discourage making

>   a new account right away.

Reviewer: wouldn't mind if some of these things were hyperlinks, but it's not important

Author: have added some - let me know if there's a better scoped place to link to.
